### ISSUE TITLE
Tflite-micro supported layers

### ISSUE THREAD

**Issue by panhu** [tensorflow/tflite-micro#1819]
Hi:
I want to ask if there are BatchNormalization layers in my model, does tflite-micro support?

**Comment by Heatdh** [comment]
Yes it does. if you look at [mutable op resolver](https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/micro_mutable_op_resolver.h you will find the list of the supported ops)
